examples/macro/BUILD.bazel

```diff
@@ -22,11 +21,15 @@ bzl_library(
     ],
 )
 
-stardoc_with_diff_test(
-    name = "mocha-docs",
-    bzl_library_target = ":mocha",
-)
-
-update_docs(
-    name = "docs",
-)
+# TODO(derek): stardoc doesn't work under bzlmod with these
+# targets when working against an overridden version of bazel-lib.
```

Comment:
Can we just not run stardoc under bzlmod?

Reply:
Only these targets, for some reason. The main docs are fine.